webapp/Connector/src/utility/MabQuery.js

```diff
@@ -714,6 +741,68 @@ Ext.define('Connector.utility.MabQuery', {
             op = ' - exclude';
         }
         return columnName + op + ": " + valueStr;
-    }
+    },
 
+    getBaseCountSQL : function(useFilter) {
+        return [
+            'SELECT',
+            [
+                'COUNT(DISTINCT mab_mix_name_std) as mixCount',
+                'COUNT(DISTINCT study) as studyCount',
+                'COUNT(DISTINCT virus) as virusCount'
+            ].join(', '),
+            this._getAssayFrom(),
+            (useFilter ? this._buildWhere(this._getMabStateFilterWhere(false, this.logging)) : '')
+        ].join('\n');
+    },
+
+    getMetaCountSQL : function(useFilter) {
+        return [
+            'SELECT COUNT(DISTINCT mab_donor_species) as donorSpeciesCount',
+            this._getMabMixMetaFrom(),
+            (useFilter ? this._buildWhere(this._getActiveMabMixIdWhere(this.logging, false, false)) : '')
+        ].join('\n');
+    },
+
+    getMabCountSQL : function(useFilter) {
+        return [
+            'SELECT COUNT(DISTINCT mab_id) as mabCount',
+            this._getMabMixMAbMetaFrom(),
+            useFilter ? this._buildWhere(this._getActiveMabMixIdWhere(this.logging, false, false)) : ''
+        ].join('\n');
+    },
+
+    getMabValuesSQL : function(useFilter) {
+        return [
+            'SELECT DISTINCT mab_name_std',
+            this._getMabMixMAbMetaFrom(),
+            useFilter ? this._buildWhere(this._getActiveMabMixIdWhere(this.logging, false, false)) : ''
+        ].join('\n');
+    },
+
+    getMAbVirusPairFrom : function(useFilter) {
+        return [
+            'FROM (SELECT ',
+            [this.MAB_MIX_NAME_STD, 'virus', 'COUNT(*) as mabVirusCount'].join(', '),
```

Comment:
mabVirusCount doesn't seem to be used anywhere, can getMAbVirusPairValuesSQL be simplified to: select distinct(mab_mix_name_std || \' - \' || virus), so that an additional inner query is not needed.